src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -95,14 +102,13 @@ public String getPattern() {
             listener.getLogger().format("Must specify user or organization%n");
             return;
         }
-        String apiUrl = null; // TODO GHE
-        StandardCredentials credentials = Connector.lookupScanCredentials(observer.getContext(), apiUrl, scanCredentialsId);
+        StandardCredentials credentials = Connector.lookupScanCredentials(observer.getContext(), null /* TODO apiUri */, scanCredentialsId);
```

Comment:
:bug: Why are you not passing `apiUri` here? It is necessary for proper domain restriction lookup.